pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/TilbakekrevingAvFeilutbetaltBeloep.kt

```diff
@@ -0,0 +1,208 @@
+                    English to "Nav will collect the tax amount from the Norwegian Tax Administration."
+                )
+            }
+            showIf(harMotregning) {
+                paragraph {
+                    text(
+                        Bokmal to "Du har i samme periode fått utbetalt for lite ",
+                        Nynorsk to "Du har i same perioden fått utbetalt for lite ",
+                        English to "In the same period, you have received too little "
+                    )
+                    includePhrase(Redigerbar.SaksType(pesysData.sakstype))
+                    text(
+                        Bokmal to "Dette er tatt med i beregningen vår.",
+                        Nynorsk to "Dette er teke med i berekninga vår.",
+                        English to "This has been taken into account in our calculations."
+                    )
+                }
+            }
+            paragraph {
+                text(
+                    Bokmal to "Vedtaket er gjort etter folketrygdloven § 22-15.",
+                    Nynorsk to "Vedtaket er gjort etter folketrygdlova § 22-15.",
+                    English to "This decision is made pursuant to the provisions of section 22-15 of the National Insurance Act."
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "I vedlegget “Oversikt over feilutbetalinger” finner du en oversikt over periodene med feilutbetalinger og beløpet du må betale tilbake.",
+                    Nynorsk to "I vedlegget “Oversikt over feilutbetalinger” finn du ei oversikt over periodane med feilutbetalingar og beløpet du må betale tilbake.",
+                    English to "The attachment titled 'Overview of Incorrect Payments' provides details on the periods with payment errors and the amounts that need to be repaid."
+                )
+            }
+            title1 {
+                text(
+                    Bokmal to "Betydning for skatteoppgjøret",
+                    Nynorsk to "Betydning for skatteoppgjeret",
+                    English to "Significance for the tax settlement"
+                )
+            }
+            paragraph {
+                text(
+                    Bokmal to "Vi rapporterer endringen til Skatteetaten. De vil korrigere skatteoppgjøret ditt ut fra denne endringen.",
+                    Nynorsk to "Vi rapporterer endringa til Skatteetaten. Dei vil korrigere skatteoppgjeret ditt ut frå denne endringa.",
+                    English to "We will report the change to the Norwegian Tax Administration. They will correct your tax settlement in view of this change.",
+                )
+            }
+            title1 {
+                text(
+                    Bokmal to "Du har rett til å klage",
+                    Nynorsk to "Du har rett til å klage",
+                    English to "You have the right to appeal"
+                )
+            }
+            paragraph {
```

Comment:
Vurder om vi kan bruke felles frasen DuHarRettTilAaKlage, eller bruk named reference for vedlegg (namedReference(vedleggDineRettigheterOgMulighetTilAaKlage))